types/luks.nix

```diff
@@ -16,6 +16,11 @@
       description = "Path to the key for encryption";
       example = "/tmp/disk.key";
     };
+    configureInitrd = lib.mkOption {
```

Comment:
The option is not really self-describing. I wonder if 'unlockInInitrd' is semantically correct.

Reply:
Perhaps `enableInitrdUnlock`?